package.json

```diff
@@ -74,15 +74,15 @@
     "coverage:report:generate": "nyc report",
     "coverage:report:open": "open-cli coverage/lcov-report/index.html",
     "coverage:report": "run-s --silent coverage:report:generate coverage:report:open",
-    "lint": "eslint \"**/*.js\"",
-    "prettier": "prettier --write \"**/*.@(js|md|json|yml)\"",
-    "prettier:check": "prettier --check \"**/*.@(js|md|json|yml)\"",
+    "lint": "eslint \"**/*.@(js|ts|tsx)\"",
```

Comment:
I've never used eslint for typescript source before, but now that I know eslint is capable of it, do you know any pros/cons of using the eslint w/the typescript parser vs. tslint? 

As long as we're linting the typescript source, I don't have a strong preference on the tooling. More a question of curiosity

Reply:
I suppose another question would be whether we're linting anything twice:  first pass eslint against the .ts/.tsx files, and then a second pass (at some point) against the emitted .js/.jsx files

Reply:
ESLint works well on TypeScript. (And sort of off topic, but Prettier works _extraordinarily_ well with TypeScript.) There's been good velocity of development of the TS–ESLint tooling in recent months and I'm pretty sure Create React App is using ESLint for TS as well as JS. I like that it's something I know (I've implemented ESLint rules) and that we don't have to adopt yet another tool.

As nothing is being emitted, it's the TS source code that is linted by this change, not the ephemerally generated JS.